src/controllers/user.controller/createUser/index.js

```diff
@@ -41,8 +41,9 @@ const createUser = catchAsync(async (req, res) => {
   await Hasura(updateUserPoints, {
```

Comment:
Please remove this invocation as well from here. Also, we will have to update the mutation to create a record in the user_points table with 0 points. update the objects in the mutation to:
`
{
      email_id: $email_id
      user_name: $user_name
      cognito_sub:$cognito_sub
      user_points: {
        data: {
          points: 0
        }
      }
    }
`